decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -265,7 +265,17 @@
         visibility: "all"
       ) do
         proposal = Decidim::Proposals::Proposal.new(params)
-        proposal.add_coauthor(participatory_space.organization)
+        coauthor = case n
+                   when 0
+                     Decidim::User.where(decidim_organization_id: participatory_space.decidim_organization_id).all.sample
```

Comment:
```suggestion
                     Decidim::User.where(decidim_organization_id: participatory_space.decidim_organization_id).order(Arel.sql('RANDOM()')).first
```
Your solution works, but as we are using Postgresql here, this solution is more efficient as it does not query all the results before picking one random result